### PR TITLE
Nick's model branch

### PR DESCRIPTION
**Pull request by rnick1**

### CODE REVIEW

File: models/Group.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    password: {
```
Comment (noheropage) [inline models/Group.js:18]: may want to add additional validators later on

File: models/Group.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        len: [8],
+      },
+    },
+    rule_id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        autoIncrement: true,
```
Comment (noheropage) [inline models/Group.js:28]: we do not want autoincrement here since these will be coming from elsewhere

File: models/Group.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      autoIncrement: true, 
+      references: {
+        model: 'user',
+        key: 'id'
+      }
+  },
+    event_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      autoIncrement: true,
```
Comment (noheropage) [inline models/Group.js:46]: same with event id; no need to autoincrement

**Review by noheropage — CHANGES_REQUESTED** [review]
If you could just remove the autoincrement of ids from foreign tables I think this would be all set!

File: models/Group.js (hunk 2 above)
Comment (rnick1) [reply models/Group.js:28]: Good point. I will delete the autoincrement

File: models/Group.js (hunk 3 above)
Comment (rnick1) [reply models/Group.js:46]: Good. Deleted.

**Pull request closed without merging**